### DISCUSSION TITLE
Use Bootstrap pagination view instead of Tailwind?

### DISCUSSION THREAD

**Discussion by enxoco** [statamic/cms#5237] [Troubleshooting] [answered]
We are in the process of migrating a client site from Statamic 2 to 3.  One of the last big issues I have been working on is pagination for an events plugin we have.  I finally tracked it down to being an issue with the pagination view being used.  It seems that by default it is using a tailwind view when I need it to use a bootstrap view instead.  Is this a setting somewhere that I can change?  For now I am passing the view name into the render function.

Is there a way to set this as a global option somewhere?  I don't think we use Tailwind anywhere else on the site anywhere.  We write all our own css.

![image](https://user-images.githubusercontent.com/34722899/153076571-d1d65203-69c9-4e00-8a20-870208ccc4d3.png)

This is what I get by default with:
```$paginator->render()```
![image](https://user-images.githubusercontent.com/34722899/153076948-540fcb47-b861-44ee-ba0d-8eda370adb50.png)

**Comment by jasonvarga** [comment, marked as answer]
Where's the first screenshot from?

**Reply by enxoco** [reply]
@jasonvarga The first screenshot is from the third party plugin we are using.  This is the code for [that](https://github.com/transformstudios/statamic-events-v3/blob/main/src/Tags/Events.php#L228)

**Reply by enxoco** [reply]
@jasonvarga Sorry I am such a noob when it comes to Laravel.  I ended up just running:

```php artisan vendor:publish --tag=laravel-pagination``` 

Then I edited the tailwind view to fit my use case.
